### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Upgrade some deps and remove unused ones

see https://github.com/ArkEcosystem/marketsquare.io/pull/2040 for details

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,30 +7,6 @@
         "name": "Brian Faust",
         "email": "hello@basecode.sh"
     }],
-    "require": {
```
Comment (ItsANameToo) [inline composer.json:10]: why did this move down? can you re-add it to the original place in the composer.json file?

**Comment by alfonsobries** [comment]
Draft until https://github.com/ArkEcosystem/laravel-stan is merged

**Comment by alfonsobries** [comment]
@ItsANameToo Just updated the composer.lock file, I am not entirely sure if that will work for you so ill explain how I was able to run  and install the composer deps:

Basically, in every project that depends on the other I  hardcoded locally (not committed) `*` as the package version and added the local path on the "repositories" section, then installed the deps with `composer install`

Just as an example see the diff below for fortify:

image

Didn't find an easy way to test that locally 😅 maybe you know another way,


Should be easy to update (or fix the issues) once the different dependencies were merged

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Comment by alfonsobries** [comment]
Closing in favor of this one https://github.com/ArkEcosystem/laravel-commonmark/pull/20

**Pull request closed without merging**